velox/dwio/parquet/tests/CMakeLists.txt

```diff
@@ -43,3 +43,6 @@ target_link_libraries(
   velox_aggregates
   velox_tpch_gen
   ${TEST_LINK_LIBS})
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/examples
```

Comment:
Wont it be better if we use something like CMake Resource for this : https://cmake.org/cmake/help/v3.13/prop_tgt/RESOURCE.html , this way CMake should track these files and take care of adding it to the right place (this will work for installs also).

Reply:
`RESOURCE` would coupled with the platform and make different directory structures dependent on the platform, say for macOS the directory would be `ExecutableTarget.app/Contents/resources/`.

Could we use `add_custom_target`, it tracks those files and takes care of adding them to the right place. I've updated this PR with it, could you help to take another look?

Reply:
`RESOURCE` is afaik (I haven't used it so far) for mac specific app bundles, so the custom target is more appropriate.